### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by Rustamxon7**
**Personal budget front end created with react 18**

All requirements are complete:

- Fetch API from Rails backend ✅
- Fetch JWT authentication and past to localhost ✅
- Use redux ✅
- Create a login page ✅
- Create a signup page ✅
- Create a signout request ✅
- Make react app ready to start the project ✅

_Feel free to review my project and let me know if any changes need to be made. 😊_

### CODE REVIEW

File: src/pages/LogIn.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/* eslint-disable react/jsx-props-no-spreading */
```
Comment (Finsoy) [inline src/pages/LogIn.js:1]: You can disable this rule in eslintrc for all project

File: src/pages/LogIn.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+const LogIn = () => {
+  const dispatch = useDispatch();
+  const { register, handleSubmit } = useForm();
+  const onSubmit = (data) => {
+    dispatch(login(data));
+  };
+
+  return (
+    <div className="">
```
Comment (Finsoy) [inline src/pages/LogIn.js:15]: Empty className

**Review by Finsoy — APPROVED** [review]
Looks cool

File: src/pages/Home.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  };
+
+  // if authtoken is not set show link to login
+
+  return (
+    <>
+      <div>
+        <h1>Home</h1>
+        <div>{isAuthenticated.data.id}</div>
+        {handleLogout}
```
Comment (Finsoy) [inline src/pages/Home.js:22]: What is it?

File: src/pages/LogIn.js (hunk 2 above)
Comment (Rustamxon7) [reply src/pages/LogIn.js:15]: I will remove it thanks.

File: src/pages/LogIn.js (hunk 1 above)
Comment (Rustamxon7) [reply src/pages/LogIn.js:1]: Thank you for the feedback!

File: src/components/NavPanel.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      text: 'Home',
+    },
+  ];
+
+  return (
+    <section>
+      <nav>
+        <ul>
+          {links.map((link) => (
+            <li key={link.id} className="">
```
[the remark this answers is not in the document]
Comment (Rustamxon7) [reply src/components/NavPanel.js:28]: Ok got it.

**Pull request merged**